app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -708,8 +713,15 @@ private void focusOnTouch(MotionEvent event) {
 
         //Calculate the Focus Area
         private Rect calculateFocusArea(float x, float y) {
-            int left = clamp(Float.valueOf((x / mSurfaceView.getWidth()) * 2000 - 1000).intValue(), FOCUS_AREA_SIZE);
-            int top = clamp(Float.valueOf((y / mSurfaceView.getHeight()) * 2000 - 1000).intValue(), FOCUS_AREA_SIZE);
+			//getting screen size
+			Display display = ((WindowManager) getContext().getSystemService(Context.WINDOW_SERVICE)).getDefaultDisplay();
+			Point size = new Point();
+			display.getSize(size);
+			screen_width = size.x;
```

Comment:
Use the functions which are already in `Utils.java` [Link](https://github.com/fossasia/phimpme-android/blob/development/app/src/main/java/vn/mbm/phimp/me/utils/Utils.java#L154-L164)